renovate-config.json

```diff
@@ -4,5 +4,21 @@
     "github>platform-engineering-org/.github",
     ":disableDependencyDashboard",
     ":pinDigestsDisabled"
+  ],
+  "packageRules": [
+    {
+      "description": "Override the docker:pinDigests preset provided by config:best-practices",
```

Comment:
Do we want to say somewhere here that `config:best-practices` is inherited from `github>platform-engineering-org/.github` and "can't" be changed?

Because if I hadn't read the PR description (which one might not have, after merge), I might wonder why we're referring to overriding an option we're not using.